packages/core/core/src/Parcel.js

```diff
@@ -385,6 +385,7 @@ export default class Parcel {
         },
       };
 
+      this.#requestTracker.flushStats();
```

Comment:
Agree this would be nicer. We opted not to do it that way to avoid adding Parcel public API that would be a breaking change to remove or modify in future. Thoughts?

Reply:
As an aside, I'm looking at adding high-level perf metrics and am starting to explore options. Having a stats object or something similar to that would be useful.

In regards to the public API, you could name the field as `unstable_stats` to make no guarantees to consumers. Or in a similar fashion, expose that there is a stats object but type it as a record with any value. That way, we are saying we will provide it but make no guarantees about what's in there. We'd probably need something internal to give us types we know about to make logs or w/e on them, but it doesn't need to be exposed.